project/Build.scala

```diff
@@ -150,8 +151,8 @@ object Build {
 
     // Avoid having to run `dotty-sbt-bridge/publishLocal` before compiling a bootstrapped project
     scalaCompilerBridgeSource :=
-      (dottyOrganization %% "dotty-sbt-bridge" % dottyVersion % Configurations.Component.name)
-      .artifacts(Artifact.sources("dotty-sbt-bridge").copy(url =
+      (dottyOrganization %% "dotty-sbt-bridge" % "NOT_PUBLISHED" % Configurations.Component.name)
```

Comment:
A change was accidentally reverted here: we stopped using NOT_PUBLISHED in https://github.com/lampepfl/dotty/commit/78ac17ace84382322aadfa18334ba2a6b214d7ad